CONTRIBUTING.rst

```diff
@@ -269,3 +275,43 @@ Name subpackages and modules using short names describing their functionality,
 e.g., ``tda`` for the subpackage containing topological data analysis work and
 ``htfa.py`` for the module implementing hierarchical topographical factor
 analysis.
+
+
+Making a release
+================
+
+This information is only of interest to the core contributors who have the
+right to make releases.
+
+1. Choose a release number, ``v``. We follow `Semantic Versioning
+   <http://semver.org>`_, although we omit the patch number when it is 0.
+
+2. Prepare the release notes::
+
+       git checkout -b release-v<v>
```

Comment:
I thought about making a script. However, it is not straightforward, because there are a couple manual steps involved (e.g., examining the generated release notes, responding to potential PR comments).

In the long term, I agree that we should use scripts as much as possible. I opened issue #249.